neard/src/cli.rs

```diff
@@ -537,11 +537,7 @@ impl RunCmd {
                 o11y_opts,
                 near_config.client_config.chain_id.clone(),
                 near_config.network_config.node_key.public_key().clone(),
-                near_config
-                    .network_config
-                    .validator
-                    .as_ref()
-                    .map(|validator| validator.account_id()),
+                near_config.network_config.validator.account_id(),
```

Comment:
Should this be mutable as well?

Reply:
I've added a TODO here
https://github.com/near/nearcore/blob/b0535005c027328c79837e600fda95f0ac23062b/chain/network/src/peer/peer_actor.rs#L310-L311
This account_id is consumed here: https://github.com/near/nearcore/blob/master/core/o11y/src/opentelemetry.rs#L45-L49
to create a name for opentelemetry channel. Changing it would require bigger changes and changing the name would make logs from single node are not visible together.